lib/components/tiled_component.dart

```diff
@@ -58,6 +59,38 @@ class TiledComponent extends Component {
         _renderLayer(c, layer);
       }
     });
+
+    map.objectGroups.forEach((objects) {
+      if (objects.visible) {
+        _renderObjectGroup(c, objects);
+      }
+    });
+  }
+
+  void _drawRotatedImageRect(Canvas canvas, src, dst, image, Offset origin, {double rotate = 0, double scaleX = 1, double scaleY = 1}) {
+    canvas.save();
+    canvas.translate( origin.dx, origin.dy );
+    canvas.rotate(rotate);
+    canvas.scale(scaleX, scaleY);
+    canvas.translate( -origin.dx, -origin.dy);
+    canvas.drawImageRect(image, src, dst, paint);
+    canvas.restore();
+  }
+
+  void _renderObjectGroup(Canvas c, ObjectGroup objects) {
+    objects.tmxObjects.forEach((tmx) {
+      if (tmx.gid != null && tmx.visible) {
+        final tile = map.getTileByGID(tmx.gid);
+        final image = images[tile.image.source];
+
+        final rect = tile.computeDrawRect();
+        final src = Rect.fromLTWH(rect.left.toDouble(), rect.top.toDouble(),
+            rect.width.toDouble(), rect.height.toDouble());
+        final dst = Rect.fromLTWH(tmx.x.toDouble(), tmx.y.toDouble()-rect.height.toDouble(),
+            rect.width.toDouble(), rect.height.toDouble());
+        _drawRotatedImageRect(c, src, dst, image, dst.bottomLeft,rotate: tmx.rotation.toDouble()*(math.pi/180));
```

Comment:
please apply dart format to this file, there some inconsistencies (eg operator spacing)